includes/class-wporg-gp-translation-events-translation-listener.php

```diff
@@ -2,55 +2,80 @@
 
 class WPORG_GP_Translation_Events_Translation_Listener {
 	const ACTIONS_TABLE_NAME = 'wp_wporg_gp_translation_events_actions';
-	const ACTION_CREATED = 'created';
+	const ACTION_CREATE = 'create';
+	const ACTION_APPROVE = 'approve';
+	const ACTION_REJECT = 'reject';
+	const ACTION_MARK_FUZZY = 'mark_fuzzy';
```

Comment:
Are we still tracking fuzzy statuses? Also, we need to track `changesrequested` status.

Reply:
`changesrequested` is now being tracked, good catch.

For tracking fuzzy, I opened #29.